### ISSUE TITLE
Player profile

### ISSUE THREAD

**Issue by toughknuckles** [Flieh/diplicity-beta-feedback#6]
In the staging screen one can see the other players,   when you click on them it doesn't do anything. 

-will you be able to view player profile?  Stats?
-will you be able to communicate before the game starts? 
-it is using google avatar....it is based off that profile?

**Comment by Flieh** [comment]
No player interaction yet.

**Comment by zond** [comment]
Yeah, I want clicking an player image to open a dialog with info. Not started on that yet though.
